.github/ISSUE_TEMPLATE/cli-issue.md

```diff
@@ -0,0 +1,16 @@
+---
+name: cli issue
+about: Create an issue to ask a question or improve the bash client.
+title: ''
+labels: cli
+assignees: mcortesi, asaj
```

Comment:
Labels and assignees should be tested. This could not be tested in own repo because users are not available.